.github/workflows/ci.yml

```diff
@@ -382,7 +382,7 @@ jobs:
           ln -s ${OPTEE_OS_TO_TEST} ${TOP}/optee_os
           cd ${TOP}/build
 
-          make -j$(nproc) check XEN_BOOT=y
+          make -j$(nproc) check XEN_BOOT=y SPMC_AT_EL=1
```

Comment:
This makes FF-A mandatory, right? Don't we want to still support (and test) the "legacy" mediator?

Reply:
Yes, it's mandatory for this test, just as it is for the Hafnium test.
I don't know how relevant the legacy mediator is.

Reply:
@lorc ?

Reply:
Well, we are still using legacy mediator in (semi-)production. So for us, at least, it is still relevant.

Frankly, I didn't followed OP-TEE development closely. Are you planning to ditch old SMC API and switch to FF-A only?

Reply:
The old SMC API is still in use and supported, but FF-A is getting more interesting as virtualization goes. We don't plan to remove anything. We could continue supporting the legacy mediator in the CI loop if we can afford it.

Reply:
> We could continue supporting the legacy mediator in the CI loop if we can afford it.

IMO, it would be ideal. But if it really stresses out CI, just test the FF-A case. I believe it is being developed more actively than the legacy API, so it is more important to test it.

Reply:
We don't reach any CI limit at the moment so we could make that a separate job and we would not notice any slowdown (assuming both take roughly the same amount of time).

Reply:
OK, let's keep the old test.